build.gradle

```diff
@@ -300,9 +300,7 @@ tasks.register('packaging', Zip) {
 defaultTasks 'clean', 'packaging'
 
 tasks.javadoc {
-    options.addStringOption('Xdoclint:none', '-quiet')
     options.addStringOption('-html5')
-    options.addBooleanOption '-no-module-directories', true
```

Comment:
Removing this flag will create wrong links in the JavaDoc search.

Reply:
added a fix